lib/include/tick/linear_model/model_hinge.h

```diff
@@ -21,19 +24,38 @@ class DLL_PUBLIC TModelHinge : public virtual TModelGeneralizedLinear<T> {
   using TModelGeneralizedLinear<T>::get_inner_prod;
   using TModelGeneralizedLinear<T>::get_class_name;
 
+ private:
+  TModelHinge() : TModelHinge<T>(nullptr, nullptr, false) {}
```

Comment:
Can we mention that is used for serialization purposes only?

Reply:
sure